src/components/learner-credit-management/data/hooks/useBudgetId.js

```diff
@@ -0,0 +1,15 @@
+import { useParams } from 'react-router-dom';
+import { isUUID } from '../utils';
+
+const useBudgetId = () => {
+  const { budgetId } = useParams();
+  const enterpriseOfferId = isUUID(budgetId) ? null : budgetId;
+  const subsidyAccessPolicyId = isUUID(budgetId) ? budgetId : null;
```

Comment:
[nit]: I think it would be helpful to add a comment for context around the differences between `enterpriseOfferId` and `subsidyAccessPolicyId`

Reply:
I agree, adding an JSDoc for `useBudgetId` would be helpful! Will update in the morning.